frontend/tsconfig.json

```diff
@@ -21,9 +16,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "baseUrl": "."
+    "baseUrl": ".",
+    "downlevelIteration": true
```

Comment:
Which part of the code required this configuration change?

Reply:
I don't remember the exact cause. But when I started to work on my personal computer, I needed to add the parameter `downlevelIteration`
I have tested it with removing that parameter and it works fine. I deleted this modification